index.html

```diff
@@ -4,23 +4,50 @@
         <title>Fruit and Veggies</title>
```

Comment:
Here is my banner for winning:

    <div ng-show="win" ng-class="{'has-success': win}">
        Congrats! You Won!
    </div>